community/contributing/index.md

```diff
@@ -8,22 +8,34 @@ redirect_from: /docs/contributing-to-spinnaker
 
 {% include toc %}
 
-## Filing issues
+Welcome to the Spinnaker community! Please join us as a contributor.
 
-Please file issues through the [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues).
+## Quick Links
 
-## How to Become a Contributor
+- [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues): file Spinnaker issues
+- [Development environments](/community/gardening/dev-environment/): set up a local development environment to test your pull requests
+- [How to submit a patch](/community/contributing/submitting/): read guidelines on submitting patches, proposing new features, and using commit and PR naming conventions
+- [Code conventions for server-side components](/community/contributing/back-end-code/): read code conventions for pull requests to core services
+- [Reviewing Pull Requests](/community/contributing/docs/reviewing-prs): get started contributing by reviewing documentation pull requests
```

Comment:
```suggestion
- [Reviewing pull requests](/community/contributing/docs/reviewing-prs): get started contributing by reviewing documentation pull requests.
```

Reply:
And because these list items are all complete sentences, go ahead and put periods after each.